File: Modules/ServerTypes/TrainingManagement/ProgramGradeType.ts
import { Decorators } from "@serenity-is/corelib";

export enum ProgramGradeType {
    Na = 0,
    PassFail = 1,
    Grade = 2,
    Score = 3
}
Decorators.registerEnumType(ProgramGradeType, 'HRMSoftware.TrainingManagement.ProgramGradeType', 'HumanResource.TrainingManagement.ProgramGradeType');

File: Modules/HumanResource/TrainingManagement/ProgramFlow/ProgramGradeType.cs
using System.ComponentModel;
using Serenity.Data;

namespace HRMSoftware.TrainingManagement;
[Serenity.ComponentModel.EnumKey("HumanResource.TrainingManagement.ProgramGradeType")]
public enum ProgramGradeType : int
{
    [Description("N/A")]
    Na = 0,
    [Description("Pass & Fail")]
    PassFail = 1,
    [Description("Grade")]
    Grade = 2,
    [Description("Score")]
    Score = 3,
}